app/index/activities/edit/route.js

```diff
@@ -75,19 +74,18 @@ export default Route.extend(RouteAutostartTourMixin, {
     this._super(...arguments)
 
     let activity = new Changeset(
-      model,
+      this.store.findRecord('activity', model.belongsTo('activity').id(), {
+        include: 'blocks'
+      }),
```

Comment:
What exactly are you trying to do here?

Reply:
This is an old version, to implement the new design, now changed.